docs/features/common_functional_options.md

```diff
@@ -1,3 +1,13 @@
+#### WithEnv
```

Comment:
Could you move it down in the file? I acknowledge that contributing to the docs is not properly documented 😞 , so atm it's my tribal knowledge that I need to sit and write down.

The first is the Image Substitutions because it goes next to the WithImage in the module's page.

L27 could be a good place, where the rest of WithXXX starts.

Sorry for causing you to rework it 🙏

Reply:
No problem, assumed alphabetical.